.github/workflows/docs.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.7]
```

Comment:
I did this because the docs build used to be very slow and I am trying to avoid building it multiple times. But this is no longer the case and I think we could make everything 3.6 and 3.7 now